<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegRW</class>
 <widget class="QDialog" name="RegRW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EngineerConcole</string>
  </property>
  <widget class="QPushButton" name="write_Mul">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Write F-B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Register Data back</string>
   </property>
  </widget>
  <widget class="mylineedit" name="addr1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="mylineedit" name="writedata">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="reg_return">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="read_Mul">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Read F-B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Register Address</string>
   </property>
  </widget>
  <widget class="mylineedit" name="returndata">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="read">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Read D-B</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="write">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Write D-B</string>
   </property>
  </widget>
  <widget class="mylineedit" name="addr2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="mylineedit" name="addr3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mylineedit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
